dashboard/app/controllers/levels_controller.rb

```diff
@@ -100,7 +100,22 @@ def update_blocks
     @level.properties[type] = blocks_xml
     @level.log_changes(current_user)
     @level.save!
-    render json: {redirect: level_url(@level)}
+    render json: {redirect: edit_level_url(@level)}
```

Comment:
Does this change the blockly editing workflow to send you to the level edit page every time you change start blocks?  That seems potentially annoying if you're iterating on something; unless I missed this, and you've unhooked the save-on-run thing.

Reply:
You're right, I should leave this as-is. I'll change it back!